cypress/e2e/partner-admin/default-tests/exportAllUsersFromOrg.cy.js

```diff
@@ -0,0 +1,30 @@
+const timeout = Cypress.env('timeout');
+
+const listOrgsUrl = '/list-orgs';
+
+function checkOrgExists() {
+  cy.get('ul > li', { timeout: timeout }).contains('Districts', { timeout: timeout }).click();
+
+  cy.get('div', { timeout: timeout }).should('contain.text', Cypress.env('testPartnerDistrictName'), {
+    timeout: timeout,
+  });
+  cy.log(`${Cypress.env('testPartnerDistrictName')} exists.`);
+}
```

Comment:
A couple of things here: 

- Technically no need to make this a separate function as we only use this in one test it would be easier/better to have this directly inside the test.

- Similar to other comments: way too many timeouts. We should remove all the timeouts that were defined here as Cypress already waits 4s by default and if these actions take longer then that's not a problem with Cypress but with the application.

- Instead of using generic selectors like `ul > li` or `div`, we should use `data-testId` attributes (preferred over `data-cy` as we can reuse the `testId` ones in Vitest): https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
This is true, we try to use data-cy as much as we can, but sometimes we don't get the right DOM element, since PrimeVue injects some html to the component

Reply:
@Emily-ejag in that case you should still use a `data-testId` at the outer level of your selector. If you do `cy.get('ul > li')` for example, Cypress will search for a `ul > li` on the page and attempt to validate against it. Same and even worse with the `cy.get('div')` as there will likely be many more of those in the DOM

Reply:
Let me double check, this is a function @ksmontville or @lucasxsong wrote, I am re-using. but I will double check

Reply:
Also I moved this function to commands :)